rocky/account/forms/account_setup.py

```diff
@@ -153,7 +153,11 @@ def save(self, **kwargs):
         if self.organization and selected_group:
             self.set_user()
             OrganizationMember.objects.get_or_create(
-                user=self.user, organization=self.organization, member_name=self.cleaned_data["name"]
+                user=self.user,
+                organization=self.organization,
+                member_name=self.cleaned_data["name"],
+                verified=True,
```

Comment:
I think it is more generic to put a default value on the OrganizationMember model itself.

Reply:
Yes it is. @dekkers pointed this out as well and continued the work on this feature.